paasta_tools/frameworks/adhoc_scheduler.py

```diff
@@ -60,3 +61,7 @@ def tasks_and_state_for_offer(self, driver, offer, state):
 
     def kill_tasks_if_necessary(self, *args, **kwargs):
         return
+
+    def validate_config(self):
```

Comment:
i don't know, the purpose of validate_config is to validate config, it just happens that we only have a single thing we want to validate atm. if we decide to extend it later - it will include more checks.